animation_nodes/__init__.py

```diff
@@ -25,7 +25,7 @@
     "author":      "Jacques Lucke",
     "version":     (2, 1, 4),
     "blender":     (2, 80, 0),
-    "location":    "Node Editor",
+    "location":    "Animation Nodes",
```

Comment:
`Animation Nodes Editor` would be more informative, I think.

Reply:
I agree if we rename the workspace the same way.

Reply:
@dragere What do you mean by workspace?

Reply:
The `Editor Type` :
![image](https://user-images.githubusercontent.com/26360562/56761092-d9507a80-679c-11e9-92ea-16fb929c7737.png)

Reply:
We can just leave it as is, we don't want to make the name needlessly long. Just change the location to `Animation Nodes Editor` and we can merge this.

Reply:
And add a period in the changelog file.

Reply:
Done in b368bbc.